File: jdroid-android-about/src/main/java/com/jdroid/android/about/AboutActivity.java
package com.jdroid.android.about;

import androidx.fragment.app.Fragment;

import com.jdroid.android.activity.FragmentContainerActivity;

public class AboutActivity extends FragmentContainerActivity {

	@Override
	protected Class<? extends Fragment> getFragmentClass() {
		return AboutAppModule.Companion.get().getAboutContext().getAboutFragmentClass();
	}

}

File: jdroid-android-about/src/main/java/com/jdroid/android/about/AboutActivity.kt
package com.jdroid.android.about

import androidx.fragment.app.Fragment

import com.jdroid.android.activity.FragmentContainerActivity

class AboutActivity : FragmentContainerActivity() {

    override fun getFragmentClass(): Class<out Fragment> {
        return AboutAppModule.get().aboutContext.getAboutFragmentClass()
    }
}
